vike/client/client-routing-runtime/prefetch.ts

```diff
@@ -138,3 +215,24 @@ async function prefetchIfPossible(url: string): Promise<void> {
   if (!(await isClientSideRoutable(pageContextFromRoute._pageId, pageContext))) return
   await prefetchAssets(pageContextFromRoute._pageId, pageContext)
 }
+
+async function prefetchContextIfPossible(url: string, expire: number | undefined): Promise<void> {
+  const now = Date.now()
+  const lastPrefetch = globalObject?.lastPrefetchTime?.get(url)
+  if (lastPrefetch && expire && now - lastPrefetch < expire) {
```

Comment:
- How about some default `expire` value? E.g. 5 seconds or maybe 30 seconds? I'm not sure.
- I wonder whether `expire` should also apply to `prefetch()` :thinking: WDYT?

Reply:
> I wonder whether expire should also apply to prefetch() 🤔 WDYT?

I think it would be reasonable to exclude the expire logic, as it is expected to be triggered less frequently than hover.

Reply:
Ok 👍

Reply:
> How about some default expire value? E.g. 5 seconds or maybe 30 seconds? I'm not sure.

For now, I set it up in 5 seconds.